content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -25,7 +25,7 @@ User can query for all Dell CSI drivers using this command:
 - This feature deploys the sdc kernel modules on all nodes with the help of an init container.
 - For non-supported versions of the OS also do the manual SDC deployment steps given below. Refer to https://hub.docker.com/r/dellemc/sdc for supported versions.
 - **Note:** When the driver is created, MDM value for initContainers in driver CR is set by the operator from mdm attributes in the driver configuration file, 
-  config.yaml. An example of config.yaml is below in this document. Do not set MDM value for initContainers in the driver CR file manually.
+  secret.yaml. An example of secret.yaml is below in this document. Do not set MDM value for initContainers in the driver CR file manually.
```

Comment:
Change to "An example of secret.yaml is provided in this document below".

Reply:
Resolved in commit 7a96e18